process/receiptsSaving.go

```diff
@@ -0,0 +1,60 @@
+package process
+
+import (
+	"bytes"
+
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go-core/data"
+	"github.com/ElrondNetwork/elrond-go-core/data/batch"
+	"github.com/ElrondNetwork/elrond-go-core/hashing"
+	"github.com/ElrondNetwork/elrond-go-core/marshal"
+	"github.com/ElrondNetwork/elrond-go/dataRetriever"
+)
+
+type marshalizedReceiptsProvider interface {
+	CreateMarshalizedReceipts() ([]byte, error)
+}
+
+// ArgsSaveReceipts holds arguments for creating a receiptsSaver
+type ArgsSaveReceipts struct {
+	MarshalizedReceiptsProvider marshalizedReceiptsProvider
+	Marshaller                  marshal.Marshalizer
+	Hasher                      hashing.Hasher
+	Store                       dataRetriever.StorageService
+	Header                      data.HeaderHandler
+}
+
+// SaveReceipts saves receipts in the storer (receipts unit)
+func SaveReceipts(args ArgsSaveReceipts) {
```

Comment:
in the future, we might convert this to a proper struct that can handle the receipts saving/loading

Might also add unit tests for this function